simulator/src/modules/Flag.js

```diff
@@ -205,16 +205,7 @@ export default class Flag extends CircuitElement {
 Flag.prototype.tooltipText =
     "FLag ToolTip: Use this for debugging and plotting.";
 Flag.prototype.helplink =
-    "https://docs.circuitverse.org/#/timing_diagrams?id=using-flags";
-
-/**
- * @memberof Flag
- * Help URL
- * @type {string}
- * @category modules
- */
```

Comment:
Why is this removed?, this is for jsdoc

Reply:
This jsdoc is for this link `Flag.prototype.helplink ="https://docs.circuitverse.org/#/miscellaneous?id=tunnel";` which is the redundant code in this file, so I removed that link, and I think If the link is removed so the jsdoc for that is also of no use, that's why I removed it . I have attached the screenshot also.

Reply:
But that will be applicable for the helplink below, So keep the jsdoc and just remove the redundant link

Reply:
Done @tachyons